components/dashboard/src/AppNotifications.tsx

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { useCallback, useEffect, useState } from "react";
+import Alert, { AlertType } from "./components/Alert";
+import dayjs from "dayjs";
+import utc from "dayjs/plugin/utc";
+import timezone from "dayjs/plugin/timezone";
+import advancedFormat from "dayjs/plugin/advancedFormat";
+import { useUserLoader } from "./hooks/use-user-loader";
+import { getGitpodService } from "./service/service";
+import { deepMerge } from "./utils";
+
+const KEY_APP_DISMISSED_NOTIFICATIONS = "gitpod-app-notifications-dismissed";
+const PRIVACY_POLICY_LAST_UPDATED = "2023-09-26";
+
+interface Notification {
+    id: string;
+    type: AlertType;
+    message: JSX.Element;
+    preventDismiss?: boolean;
+    onClose?: () => void;
+}
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+dayjs.extend(advancedFormat);
+
+export function localizedTime(dateStr: string): JSX.Element {
+    const formatted = dayjs.utc(dateStr).local().format("dddd, MMM. D, HH:mm (z)");
```

Comment:
probably a good candidate to wrap w/ `useMemo()`

Reply:
Removed the functions entirely, because we ended up not using them 🙏